src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -138,6 +150,14 @@ def _modular_nav(
             info = self._switch_to_next_skill(e=0, info=info)
         return action, info
 
+    def _rl_nav(self, obs: Observations, info: Dict[str, Any]) -> Tuple[DiscreteNavigationAction, Any]:
+        action, term = self.nav_to_obj_agent.act(obs)
+        if term:
+            action = DiscreteNavigationAction.NAVIGATION_MODE
+            self._switch_to_next_skill(e=0, info=info)
```

Comment:
Right now that's how it is for all skill switches. Since modular nav can't be vectorized, I'm guessing this has been hardcoded to 0 everywhere for now.

Reply:
can we make this a constant or something?
modular nav IS vectorized actually, it has a for loop

Reply:
actually don't worry about making it a constant. just add a comment or something

Reply:
Added a comment